packages/smooth_app/lib/pages/product/confirm_and_upload_picture.dart

```diff
@@ -215,3 +166,34 @@ class _ConfirmAndUploadPictureState extends State<ConfirmAndUploadPicture> {
     await model.onCreateProduct(barcode); // TODO(monsieurtanuki): a bit fishy
   }
 }
+
+/// Standard button for this page.
+class _OutlinedButton extends StatelessWidget {
```

Comment:
I'n not a fan of having different private buttons per page can't we use one we already have

Reply:
Before in this page the same code was repeated 3 times (once for each button), so it's an improvement, isn't it?

That said, feel free to create an issue about uniformisation of buttons in pages: probably not all identical, but at least "classes" of buttons that can be reused.